pipeline/builder/builder.go

```diff
@@ -168,6 +187,99 @@ func (b *Builder) buildRunJobStage(index int, s config.Stage) (*types.RunJobStag
 	return rjs, nil
 }
 
+func (b *Builder) buildV2ManifestStage(index int, s config.Stage) (*types.ManifestStage, error) {
+	ds := &types.ManifestStage{
+		StageMetadata:           buildStageMetadata(s, "deployManifest", index, b.isLinear),
+		Account:                 s.Account,
+		CloudProvider:           "kubernetes",
+		Location:                "",
+		ManifestArtifactAccount: "embedded-artifact",
+		ManifestName:            "",
+		Moniker: types.Moniker{
+			App:     b.pipeline.Application,
+			Cluster: fmt.Sprintf("%s-%s", b.pipeline.Application, s.Account),
+			Detail:  "",
```

Comment:
We should probably support detail and stack right?

Reply:
yep, had some tunnel vision on this, fixed